apps/tailwind-components/README.md

```diff
@@ -57,33 +57,33 @@ git switch -c <my-branch>
 
 ### Setup
 
-We use [yarn workspaces](https://yarnpkg.com/features/workspaces) to manage dependencies across all applications. On the first time, install the dependencies.
+We use [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces) to manage dependencies across all applications. On the first time, install the dependencies.
 
 ```bash
-yarn
+npm
 ```
 
-If you would like to add one or more libraries, install them using the `yarn add` method.
+If you would like to add one or more libraries, install them using the `npm add` method.
 
 ```bash
-yarn add <package-name> # add the -D flag if needed
+npm add <package-name> # add the -D flag if needed
```

Comment:
Consider replacing `add` with `install`, as the former is only an alias of the latter: https://docs.npmjs.com/cli/v8/commands/npm-install